msal/broker.py

```diff
@@ -0,0 +1,202 @@
+"""This module is an adaptor to the underlying broker.
+PyMsalRuntime is the package providing broker's functionality.
+
+PyMsalRuntime currently supports these Windows versions, listed in this MSFT internal link
+https://github.com/AzureAD/microsoft-authentication-library-for-cpp/pull/2406/files
+"""
+from threading import Event
+import json
+import logging
+import uuid
+
+import pymsalruntime  # ImportError would be raised on unsupported platforms such as Windows 8
+    # Its API description is available in site-packages/pymsalruntime/PyMsalRuntime.pyi
+
+logger = logging.getLogger(__name__)
```

Comment:
>logger [](http://example.com/codeflow?start=0&length=6)

Any plans to integrate this logger with the msalruntime logging? We have a logging callback

Reply:
Just an FYI the logger is VERY verbose.... something to consider

Reply:
> > logger
> 
> Any plans to integrate this logger with the msalruntime logging? We have a logging callback

Definitely. The initial "release note" message of pymsalruntime 0.5.0 did not mention logging callback, so I got that impression that logging callback were not yet available. Now I realize that I am unblocked. I'll work on it.



> Just an FYI the logger is VERY verbose.... something to consider

Hopefully you use different log levels wisely. :-)